Test/run_scalability_tests.py

```
@@ -0,0 +1,442 @@
+        """최신 결과 파일들 찾기"""
+        current_time = datetime.now()
+        result_files = []
+        
+        # 최근 10분 내 생성된 파일들 찾기
+        for pattern in ['performance_results_*.json', 'performance_test_*.csv']:
+            for file_path in Path('.').glob(pattern):
+                try:
+                    file_time = datetime.fromtimestamp(file_path.stat().st_mtime)
+                    if (current_time - file_time).total_seconds() < 600:  # 10분
+                        result_files.append(str(file_path))
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)